allure-framework-modules/mrchecker-database-module/src/main/java/com/capgemini/ntc/database/core/BaseDatabase.java

```diff
@@ -0,0 +1,130 @@
+package com.capgemini.ntc.database.core;
+
+import com.capgemini.ntc.database.core.base.properties.PropertiesFileSettings;
+import com.capgemini.ntc.database.core.base.runtime.RuntimeParameters;
+import com.capgemini.ntc.test.core.BaseTest;
+import com.capgemini.ntc.test.core.BaseTestWatcher;
+import com.capgemini.ntc.test.core.ITestObserver;
+import com.capgemini.ntc.test.core.ModuleType;
+import com.capgemini.ntc.test.core.analytics.IAnalytics;
+import com.capgemini.ntc.test.core.base.environment.IEnvironmentService;
+import com.capgemini.ntc.test.core.base.properties.PropertiesSettingsModule;
+import com.capgemini.ntc.test.core.logger.BFLogger;
+import com.google.inject.Guice;
+
+import lombok.Getter;
+import ru.yandex.qatools.allure.annotations.Attachment;
+
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+
+abstract public class BaseDatabase implements ITestObserver {
+
+	private final static PropertiesFileSettings propertiesFileSettings;
+	private static       IEnvironmentService    environmentService;
+	private final static IAnalytics             analytics;
```

Comment:
If change access here to public, then it could be accessed exactly the same way as it happens in getAnalytics(). Final will guarantee unchangeable.